spring-cloud-starter-stream-sink-gemfire/src/test/java/org/springframework/cloud/stream/app/gemfire/sink/normal/GemfireSinkIntegrationTests.java

```diff
@@ -126,6 +98,7 @@ public void testByteArrayJsonPayload() {
 			assertThat(transformer.toString(region.get("key")), equalTo(messageBody));
 		}
 
+		@Ignore
```

Comment:
After un`@Ignore` these test and running together with the `GemfireSinkIntegrationTests` I get this exception:
```
Caused by: org.apache.geode.cache.client.ServerOperationException: remote server on 10.0.0.134(11316:loner):62744:7a355dcf: org.apache.geode.security.AuthenticationRequiredException: Server expecting SSL connection
	at org.apache.geode.cache.client.internal.pooling.ConnectionManagerImpl.borrowConnection(ConnectionManagerImpl.java:246)
	at org.apache.geode.cache.client.internal.OpExecutorImpl.execute(OpExecutorImpl.java:148)
	at org.apache.geode.cache.client.internal.OpExecutorImpl.execute(OpExecutorImpl.java:127)
	at org.apache.geode.cache.client.internal.PoolImpl.execute(PoolImpl.java:782)
	at org.apache.geode.cache.client.internal.PutAllOp.execute(PutAllOp.java:100)
	at org.apache.geode.cache.client.internal.ServerRegionProxy.putAll(ServerRegionProxy.java:604)
	at org.apache.geode.internal.cache.LocalRegion.basicPutAll(LocalRegion.java:9515)
	at org.apache.geode.internal.cache.LocalRegion.putAll(LocalRegion.java:9448)
	at org.apache.geode.internal.cache.LocalRegion.putAll(LocalRegion.java:9460)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.springframework.data.gemfire.GemfireTemplate$RegionCloseSuppressingInvocationHandler.invoke(GemfireTemplate.java:534)
	at com.sun.proxy.$Proxy94.putAll(Unknown Source)
	at org.springframework.integration.gemfire.outbound.CacheWritingMessageHandler$1.doInGemfire(CacheWritingMessageHandler.java:96)
	at org.springframework.data.gemfire.GemfireTemplate.execute(GemfireTemplate.java:459)
	... 65 more
```

I believe you get something similar, but still it fully unclear to me what is wrong. Something like we have a server in SSL mode, but our client connects without 🤷‍♂️

Reply:
You can run both tests one by one, but apparently not both together. 
They obviously interfere and i was not able to track the cause of it. Furthermore the existing geode integration testing infrastructure looks bit outdated and unnecessarily complicated. I believe we can improve this so i created https://github.com/spring-cloud-stream-app-starters/gemfire/issues/19.